.gitlab-ci.yml

```diff
@@ -107,20 +109,17 @@ test-dylint:
   stage:                           test
   <<:                              *docker-env
   script:
+    # We need to fix this test to a toolchain because of the UI tests
+    # We can't use stable here because the UI tests freak out if there are dots in the drivers file name
```

Comment:
When `cargo dylint` builds its driver it integrates the used toolchain's name into the filename of the library. Fixed stable toolchains contain dots "1.63.0" instead of dashes like for nightly "nightly-2022-06-30". The UI test framework does not like this at all. It just errors out with "invalid chars in lib name"  or something like that.